docs/reference/search/field-caps.asciidoc

```diff
@@ -1,70 +1,108 @@
 [[search-field-caps]]
 === Field Capabilities API
 
-The field capabilities API allows to retrieve the capabilities of fields among multiple indices.
-
-The field capabilities API by default executes on all indices:
+Allows you to retrieve the capabilities of fields among multiple indices.
 
 [source,console]
 --------------------------------------------------
 GET _field_caps?fields=rating
 --------------------------------------------------
 
-The request can also be restricted to specific indices:
 
-[source,console]
---------------------------------------------------
-GET twitter/_field_caps?fields=rating
---------------------------------------------------
-// TEST[setup:twitter]
+[[sample-api-request]]
```

Comment:
Looks like the placeholder anchors are used throughout the file. Need to replace those with something like `[[field-capabilities-api-...]]`

Reply:
:o Sorry, I was blind to it.